charts/xchemlab/values.yaml

```
@@ -0,0 +1,397 @@
+    password:
+      secretName: postgres-passwords
+      secretKey: password
+  opa:
+    serviceSuffix: opa-kube-mgmt
+
+grafana:
+  enabled: true
+
+  rbac:
+    create: false
+
+  serviceAccount:
+    create: false
+    name: default-full-access-mounted
+
+  ingress:
+    enabled: true
+    hosts:
+      - xchemlab-grafana.diamond.ac.uk
+    path: /
+    tls:
+      - hosts:
+          - xchemlab-grafana.diamond.ac.uk
+
+  grafana.ini:
+    server:
+      domain: xchemlab-grafana.diamond.ac.uk
+      root_url: https://xchemlab-grafana.diamond.ac.uk
+      serve_from_sub_path: true
+    auth.generic_oauth:
+      enabled: true
+      auto_login: true
+      client_id: $__file{/etc/secrets/oauth_provider/client-id}
+      client_secret: $__file{/etc/secrets/oauth_provider/client-secret}
+      scopes: openid profile email offline_access
+      auth_url: https://authbeta.diamond.ac.uk/cas/oidc/authorize
+      token_url: https://authbeta.diamond.ac.uk/cas/oidc/accessToken
+      api_url: https://authbeta.diamond.ac.uk/cas/oidc/profile
+      use_pkce: true
+      use_refresh_token: true
+      email_attribute_path: id
+
+  extraSecretMounts:
+    - name: oauth-provider
+      secretName: oauth-provider
+      defaultMode: 0440
+      mountPath: /etc/secrets/oauth_provider
+      readOnly: true
+
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+        - name: Prometheus
+          type: prometheus
+          url: https://xchemlab-thanos.diamond.ac.uk
+          isDefault: true
+
+keda:
+  enabled: false
+
+opa-kube-mgmt:
+  enabled: true
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/path: "/metrics"
+    prometheus.io/port: "8182"
+  mgmt:
+    enabled: false
+  prometheus:
+    enabled: true
+  serviceAccount:
+    create: false
+  rbac:
+    create: false
+
+prometheus:
+  enabled: true
+
+  kube-state-metrics:
+    enabled: false
+
+  prometheus-node-exporter:
+    enabled: false
+
+  prometheus-pushgateway:
+    enabled: false
+
+  rbac:
+    create: false
+
+  alertmanager:
+    serviceAccount:
+      create: false
+      name: default-full-access-mounted
+    ingress:
+      enabled: true
+      hosts:
+        - host: xchemlab-alertmanager.diamond.ac.uk
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+      tls:
+        - hosts:
+            - xchemlab-alertmanager.diamond.ac.uk
+      annotations:
+        nginx.ingress.kuberentes.io/use-regex: "true"
+        nginx.ingress.kubernetes.io/auth-url: https://xchemlab-alertmanager.diamond.ac.uk/oauth2/auth
+        nginx.ingress.kubernetes.io/auth-signin: https://xchemlab-alertmanager.diamond.ac.uk/oauth2/start?rd=$escaped_request_uri
+    extraArgs:
+      web.external-url: https://xchemlab-alertmanager.diamond.ac.uk
+    configmapReload:
+      enabled: true
+
+  server:
+    remoteWrite:
+      - url: http://test-thanos-receive.xchemlab.svc.cluster.local:19291/api/v1/receive
```

This should be replaced as below:
```suggestion
      - url: http://{{ .Release.name }}-thanos-receive.xchemlab.svc.cluster.local:19291/api/v1/receive
```
Awaiting https://github.com/prometheus-community/helm-charts/pull/3777